backend-server/collections/games.js

```diff
@@ -0,0 +1,102 @@
+const { turnPhases } = require("../models/turn");
```

Comment:
Be careful, not sure if this branch is up to date with my latest changes, but I refactored this file into the Game Class itself.

Reply:
I'm pretty sure I have not merged to this branch since I started it over a week ago. Will make sure to merge carefully and test.